### PR TITLE
app-misc/qstamina: new package

### PR DESCRIPTION
**Pull request by chelovechishko**

### CODE REVIEW

**Comment by monsieurp** [comment]
Hi. Thanks. Please read https://wiki.gentoo.org/wiki/Gentoo_Github on how to properly commit when making a PR. I can tell you've used git without resorting to `repoman`. Also pls detail the commit "New package" is obvious since it's a package, doesn't it.

File: app-misc/qstamina/Manifest [hunk 1]
```diff
@@ -0,0 +1,4 @@
+AUX qstamina-src.patch 678 SHA256 89ca029a70c1078bb92b5f93f4b70eb6b5de9fd9d19287823b0f8d680d1087ba SHA512 b77fd5d752dbe03c41b291e7044b2e14f5c253c8466bde794757425d1d24d9612b24651ab9172a68d1458c4629a6a0ecbf7ee8c766124b162e954a7576a73b17 WHIRLPOOL 82340adfcdf8a6ff4e52600a2b20a29654f2ef70a46d0de7e469a321e1cd63295a4a67be9172c95f8de3a16a2f84d377063b04ca0fff16fefa3213f5121b9151
```
Comment (monsieurp) [inline app-misc/qstamina/Manifest]: Still not correct. Are you using `repoman commit` to perform the commit?

File: app-misc/qstamina/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```
Comment (monsieurp) [inline app-misc/qstamina/metadata.xml]: There are missing information. Please take a look at https://github.com/gentoo/gentoo/blob/master/sys-libs/argp-standalone/metadata.xml for example.

File: app-misc/qstamina/qstamina-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	epatch "${FILESDIR}/qstamina-stamina.patch"
```
Comment (SoapGentoo) [inline app-misc/qstamina/qstamina-9999.ebuild]: Don't use `epatch` anymore please, put the patches in a bash array called `PATCHES` and then completely remove the `src_prepare` function.

File: app-misc/qstamina/qstamina-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+	epatch "${FILESDIR}/qstamina-src.patch"
+	default
+}
+
+src_configure() {
+	eqmake5
+}
+
+S="${WORKDIR}/${P}/src"
```
Comment (SoapGentoo) [inline app-misc/qstamina/qstamina-9999.ebuild]: put this after `RDEPEND` please

**Comment by monsieurp** [comment]
ping?

**Comment by monsieurp** [comment]
Feel free to reopen this PR once those errors are dealt with. Cheers.

**Pull request closed without merging**